hazelcast/src/test/resources/com/hazelcast/config/empty-object.yaml

```diff
@@ -0,0 +1,13 @@
+hazelcast:
+  network:
+    public-address: 127.0.0.1
+    port:
+      auto-increment: 5
+      port-count: 100
+      port: 5701
+    reuse-adress: false
+    join:
+      auto-detection:
+        enabled: true
+    memcache-protocol:
+    enabled: ture
```

Comment:
Maybe the file name could reflect this failure reason.

Reply:
The failure reason is the empty `memcache-protocol` object. I changed `ture` to `true` to make it less distacting.